docs/index.asciidoc

```diff
@@ -70,13 +70,13 @@ NOTE: The `retry_failed` option does not control the library level retry.
 
 This plugin supports the following configuration options plus the <<plugins-{type}s-{plugin}-common-options>> described later.
 
+NOTE: As of version `6.0.0` of this plugin, a number of previously deprecated settings related to SSL have been removed. Please see the
+<<plugins-{type}s-{plugin}-obsolete-options>> for more details.
```

Comment:
```suggestion
NOTE: As of version `6.0.0` of this plugin, a number of previously deprecated settings related to SSL have been removed. 
Please check out <<plugins-{type}s-{plugin}-obsolete-options>> for details.
```

Reply:
Asciidoc best practice is to start each sentence on a new line. The modular approach makes updating docs source easier to review and change.